.github/workflows/test.yaml

```diff
@@ -8,6 +8,18 @@ on:
 jobs:
   tests:
     runs-on: ubuntu-22.04
+    services:
+      postgres:
+        image: postgres
```

Comment:
Don't use `:latest`! Pin latest stable (`15.3-bullseye`) so there's no question about what runs on the CI (when pb occurs)